tests/unit/amo/components/TestCategories.js

```diff
@@ -70,14 +70,14 @@ describe('<Categories />', () => {
       result: [
         {
           application: 'android',
-          name: 'Games',
-          slug: 'Games',
+          name: 'Travel',
```

Comment:
You should leave this test how it is and add a few out-of-order categories, making a new test to ensure sorting works how you want.